plugins/services/src/js/components/modals/CreateServiceModal.js

```diff
@@ -491,6 +497,32 @@ class CreateServiceModal extends Component {
     this.setState({ isPending: true });
   }
 
+  getExpandAdvancedSettings() {
+    // If we have errors inside the advanced options, expand the options to allow the user to see the errors.
+    const errors = this.anyCriticalFormErrors();
+
+    if (errors.length > 0) {
```

Comment:
`anyCriticalFormErrors` used to return boolean, right? If we change what it returns we should also change the name and JSDoc

Reply:
I changed the name, I can't find the JSDoc for this file.

Reply:
Done.